src/components/Widgets/MarkdownControlElements/RawEditor/index.js

```diff
@@ -322,27 +321,26 @@ export default class RawEditor extends React.Component {
       onDragOver={this.handleDragOver}
       onDrop={this.handleDrop}
     >
-      <Toolbar
-        isOpen={showToolbar}
-        selectionPosition={selectionPosition}
-        onH1={this.handleHeader('#')}
-        onH2={this.handleHeader('##')}
-        onBold={this.handleBold}
-        onItalic={this.handleItalic}
-        onLink={this.handleLink}
-        onToggleMode={this.handleToggle}
-      />
-      <BlockMenu
-        isOpen={showBlockMenu}
-        selectionPosition={selectionPosition}
-        plugins={plugins}
-        onBlock={this.handleBlock}
-        onAddAsset={onAddAsset}
-        onRemoveAsset={onRemoveAsset}
-        getAsset={getAsset}
-      />
-      <textarea
-        ref={this.handleRef}
+      <Sticky className={styles.editorControlBar} fillContainerWidth>
+        <Toolbar
+          selectionPosition={selectionPosition}
+          onH1={this.handleHeader('#')}
+          onH2={this.handleHeader('##')}
+          onBold={this.handleBold}
+          onItalic={this.handleItalic}
+          onLink={this.handleLink}
+          onToggleMode={this.handleToggle}
+          rawMode={rawMode}
+          plugins={plugins}
+          onSubmit={this.handlePluginSubmit}
+          onAddAsset={onAddAsset}
+          onRemoveAsset={onRemoveAsset}
+          getAsset={getAsset}
+          rawMode
```

Comment:
The prop is duplicated here, so it'll always be true. I think we should remove the above line that goes `rawMode={rawMode}` along with the `rawMode` prop on the `RawEditor`.